src/index.html

```diff
@@ -68,6 +69,11 @@ <h2>Jumbotron</h2>
             <p>Build around the jumbotron with a navbar and some basic grid columns.</p>
             <p><a class="btn btn-secondary" href="./templates/jumbotron/index.html" role="button">View details »</a></p>
           </div>
+          <div class="col-md-4">
+            <h2>Navbar</h2>
+            <p>A navigation bar is a navigation header that is placed at the top of the page</p>
+            <p><a class="btn btn-secondary" href="./templates/navbar/index.html" role="button">View details >></a></p>
```

Comment:
Could you switch out `>>` for `»` after "View details"

Reply:
I´ve changed symbol to »